addons/im_livechat/static/src/views/livechat_form_renderer.js

```diff
@@ -0,0 +1,42 @@
+import { Discuss } from "@mail/core/public_web/discuss";
+
+import { onMounted, onWillStart, onWillUpdateProps, useState } from "@odoo/owl";
+
+import { useService } from "@web/core/utils/hooks";
+import { FormRenderer } from "@web/views/form/form_renderer";
+
+export class LivechatSessionFormRenderer extends FormRenderer {
+    static template = "im_livechat.LivechatDiscuss";
+    static components = {
+        ...FormRenderer.components,
+        Discuss,
+    };
+
+    setup() {
+        super.setup();
+        this.store = useState(useService("mail.store"));
+        onWillStart(async () => {
+            this.thread = await this.restoreDiscussThread(this.props);
+        });
+        onWillUpdateProps(async (nextProps) => {
+            this.thread = await this.restoreDiscussThread(nextProps);
+        });
+        onMounted(() => (this.store.discuss.isActive = false));
```

Comment:
Rather than overriding the other value here, it would be nice the original value was only set when necessary (so in "discuss_client_action" rather than discuss component).